server/infrastructure/saved_players_repository.py

```
@@ -0,0 +1,111 @@
+                detail="Firebase is not configured"
+            )
+        
+        try:
+            players_ref = self.db.collection('users').document(user_id).collection('saved_players').stream()
+            saved_players = []
+            
+            for player_doc in players_ref:
+                player_data = player_doc.to_dict()
+                saved_players.append(SavedPlayer(**player_data))
+            
+            return saved_players
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)